paper-api/src/main/java/org/bukkit/block/CreakingHeart.java

```diff
@@ -1,7 +1,60 @@
 package org.bukkit.block;
 
+import org.bukkit.Location;
+import org.bukkit.entity.Creaking;
+import org.jspecify.annotations.NullMarked;
+import org.jspecify.annotations.Nullable;
+
 /**
  * Represents a captured state of a creaking heart.
  */
+@NullMarked
 public interface CreakingHeart extends TileState {
+
+    /**
+     * Gets the creaking protecting this creaking heart.
+     *
+     * @return the creaking or null if this creaking heart don't have protector.
+     */
+    @Nullable
+    Creaking getCreaking();
```

Comment:
Anything against sticking to the vanilla naming here? E.g. `getProtector`?
Other peoples input here before actually changing this tho.

Reply:
Get creaking makes more sense than protector, because people wouldn't assume that the creaking is "protecting" the heart but instead is spawned by it.